manifests/j/Joplin/Joplin/2.10.17/Joplin.Joplin.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=QUSU.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Joplin.Joplin
+PackageVersion: 2.10.17
+PackageLocale: en-US
+Publisher: Laurent Cozic
+PublisherUrl: https://github.com/laurent22/joplin
+PublisherSupportUrl: https://github.com/laurent22/joplin/issues
+PrivacyUrl: https://github.com/laurent22/joplin#privacy
+Author: Laurent Cozic
+PackageName: Joplin
+PackageUrl: https://joplinapp.org
+License: MIT License
```

```suggestion
License: AGPL-3.0-or-later
```
According to LicenseUrl

---

Sorry, I just updated what was there. Good catch, thanks! Updated!